benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/BenchmarkService.java

```diff
@@ -67,28 +73,30 @@ private BenchmarkService(final String filePath) throws Exception {
 
   /**
    * Function to run all procedures related to benchmarking.
+   *
    * @throws Exception Thrown if benchmarking fails.
    */
   private void runBenchmarks() throws Exception {
     final List<BenchmarkResults> results = new ArrayList<>();
-    for (Double sf: scaleFactors) {
+    for (Double sf : scaleFactors) {
       DataGenerator.generateData(benchmarkPath, sf);
-      QueryGenerator.generateQueries(benchmarkPath, sf);
-      for (final String type: types) {
+      // QueryGenerator.generateQueries(benchmarkPath, sf);
```

Comment:
Why is this commented out?

Reply:
this part gets tpch qgen generated queries but all are unsupported currently